src/Model/Join.php

```diff
@@ -250,22 +250,6 @@ public function addFields($fields = [])
         return $this;
     }
 
-    /**
-     * Adds any object to owner model.
-     */
-    public function add(object $object, array $defaults = []): object
```

Comment:
@georgehristov it turned out, that this method is completely unused, so I removed it.